### DISCUSSION TITLE
Select languages for pull request?

### DISCUSSION THREAD

**Discussion by nclm** [WeblateOrg/weblate#12499] [Q&A]
Hi! Is there a way to select which languages to include when pushing changes from Weblate to create a git PR? Right now it seems to automatically include everything, include languages that are still work in progress. I cannot figure out how to only select the languages I want. Thanks :)

**Comment by nijel** [comment]
No, see https://github.com/WeblateOrg/weblate/issues/5563. https://github.com/WeblateOrg/weblate/issues/3436 might be also related.
